src/types/Offer.ts

```diff
@@ -138,6 +138,9 @@ type BaseOffer = {
 
 type Offer = Place | Event;
 
+export const usesLegacyLocation = (offer) =>
```

Comment:
```suggestion
export const hasLegacyLocation = (offer) =>
```

The uses is a bit confusing, because it looks like a hook

Reply:
Yeah very true